mod/bootstrap/set/all/bootstrap/navbar.rb

```diff
@@ -40,6 +40,10 @@ def navbar_responsive id, content, opts, nav_opts
     end
   end
 
+  # Generates HTML markup for a breadcrumb trail.
+  # @param items [list]: A list of items representing the breadcrumb trail.
+  # Returns:
+  # str: HTML markup for the breadcrumb trail.
```

Comment:
Yardoc (which we use to generate, eg docs.decko.org) understands @param (yay!) and @return

```
@return [String] HTML markup...
```